packages/main/bundle.esm.js

```diff
@@ -13,6 +13,7 @@ import "./dist/json-imports/LocaleData.js"
 
 import "./dist/features/InputElementsFormSupport.js";
 import "./dist/features/InputSuggestions.js";
+import "./dist/features/ShellBarAnimatedCoPilot.js";
```

Comment:
Can we call this `CoPilotAnimation.js`, as it might be used without the shell bar and sounds more like the other features (noun without adjective)

Reply:
Done.